blueprints/data-solutions/cmek-via-centralized-kms/variables.tf

```diff
@@ -12,28 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-variable "billing_account" {
-  description = "Billing account id used as default for new projects."
-  type        = string
-}
-
 variable "location" {
   description = "The location where resources will be deployed."
   type        = string
   default     = "europe"
 }
 
-variable "project_kms_name" {
-  description = "Name for the new KMS Project."
+variable "prefix" {
+  description = "Optional prefix used to generate resources names."
   type        = string
-  default     = "my-project-kms-001"
+  nullable    = false
 }
 
-variable "project_service_name" {
-  description = "Name for the new Service Project."
-  type        = string
-  default     = "my-project-service-001"
+variable "project_create" {
```

Comment:
maybe combine in `project_config` and leverage `optional`?

Reply:
I like the idea, implemented.